Automattic/sensei
# Add the "Dev" changelog section

## Proposed Changes

* Add the "Dev" section to the changelog.
* Add more detailed descriptions of each change log section.
* Fix the editorconfig for the composer.json file so it uses tabs instead of spaces.

## Testing Instructions


1. Run `npm run changelog`.
2. For the change significance step, select one of `patch/minor/major`.
3. In the type of change step, make sure there is a `dev` section.

## Pre-Merge Checklist
